mysql-test/suite/binlog/t/binlog_gtid_recovery.test

```diff
@@ -0,0 +1,41 @@
+# PS-9132 mysql.gtid_executed persistent GTID info lost when MySQL crash in Gtid_state::save
+#
+--source include/have_debug.inc
+--source include/have_log_bin.inc
+--source include/have_gtid.inc
+
+RESET MASTER;
+CREATE TABLE t1 (c1 INT NOT NULL PRIMARY KEY) ENGINE=InnoDB;
+
+SHOW BINARY LOGS;
+SELECT * FROM mysql.gtid_executed;
+
+--echo Kill the server
+--source include/kill_mysqld.inc
+
+--echo Restart the server only to kill it before GTID is saved during recovery process
+--error 137
```

Comment:
Any chance there is a mnemonic for `137`?

Reply:
I couldn't find one.